main.go

```diff
@@ -76,6 +77,11 @@ func main() {
 			Usage:  "remove old images after updating",
 			EnvVar: "WATCHTOWER_CLEANUP",
 		},
+		cli.BoolFlag{
```

Comment:
I think it would be nice to add an urfave command `monitor-only` instead of a flag, what do you think?